tools/cli/adminDBScanCommand.go

```diff
@@ -125,29 +125,14 @@ func checkExecution(
 	invariants []executions.InvariantFactory,
 	fetcher executions.ExecutionFetcher,
 ) (interface{}, invariant.ManagerCheckResult) {
-	client, session := connectToCassandra(c)
-	defer session.Close()
-	logger := loggerimpl.NewNopLogger()
+	execManager := initializeExecutionStore(c, common.WorkflowIDToHistoryShard(req.WorkflowID, numberOfShards), 0)
```

Comment:
not related to your change but looks like we can do the initialization only once for the db scan/clean command (also also provide an rps limit) instead of creating a new session for each workflow record. Would you mind creating an issue for it?